recipes/gtotree/post-link.sh

```diff
@@ -0,0 +1,4 @@
+printf "will cite" | parallel --citation 2&> /dev/null
+
+# removing coreutils ls so the user-system one is used still
+rm ${CONDA_PREFIX}/bin/ls
```

Comment:
Is this strictly needed? We really try to avoid modifying other packages.

Reply:
Heya,

Yea, it ruins people's ls settings if I don't remove it. I need some things i can easily get from the coreutils package, i just don't want to add another ls program in front of the user's regular one.